mozilla-services/consvc-shepherd
# [SHEP-153] Removing redudant env variable definition for local dev

## References

[SHEP-153](https://mozilla-hub.atlassian.net/browse/SHEP-153)

## Problem Statement

This env variable`DJANGO_SETTINGS_MODULE` is a redundant definition and can cause things to pass locally that might fail in prod/stage. 

## Proposed Changes

Removing this env variable.

## Verification Steps

Verify application works as normally expected, tests pass, and scripts run.

[SHEP-153]: https://mozilla-hub.atlassian.net/browse/SHEP-153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
